foundation/foundation-upgrade/src/test/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/OrphedomosToFabric8MigrationSteps.java

```diff
@@ -0,0 +1,88 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractMigrationTest;
+import com.boozallen.aissemble.upgrade.util.pom.PomHelper;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.apache.maven.model.Model;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertNotNull;
+
+public class OrphedomosToFabric8MigrationSteps  extends AbstractMigrationTest {
+
+    @Given("A pom with Orphedomos in the plugin management")
+    public void a_pom_with_orphedomos_in_the_plugin_management() {
+        testFile = getTestFile("v1_8_0/OrphedomosToFabric8Migration/migrate-configuration/pom.xml");
+    }
+    @When("The migration executes")
+    public void the_migration_executes() {
+        performMigration(new OrphedomosToFabric8Migration());
+        assertTrue(shouldExecute);
+    }
+    @Then("The pom is updated to use the Fabric8 docker-maven-plugin")
+    public void the_pom_is_updated_to_use_the_fabric8_docker_maven_plugin() {
+        Model model = PomHelper.getLocationAnnotatedModel(testFile);
+        Object artifactId = model.getBuild().getPlugins().get(1).getArtifactId();
+        assertEquals("docker-maven-plugin",artifactId.toString());
+
+        Object groupId = model.getBuild().getPlugins().get(1).getGroupId();
+        assertEquals("io.fabric8",groupId.toString());
+
+    }
+
+    @And("Fabric8 is configured properly")
+    public void fabric8_is_configured_properly(){
+        Model model = PomHelper.getLocationAnnotatedModel(testFile);
+        Object executions = model.getBuild().getPlugins().get(1).getExecutions();
+        assertNotNull(executions);
```

Comment:
A: ideally we would like for all of our assert calls to have an error message

Reply:
added messages for `assertNotNull`, `assertNull`, `assertTrue`, `assertFalse` calls